SF_Users_API/db/migrations/20171030225656_initial.js

```diff
@@ -0,0 +1,33 @@
+
+exports.up = function(knex, Promise) {
+  return Promise.all([
+    knex.schema.createTable('sf_users', function(table) {
+      table.increments('id').primary();
+      table.string('username');
+      table.string('email');
+      table.string('address');
+
+      table.timestamps(true, true);
+    }),
+
+    knex.schema.createTable('sf_favorites', function(table) {
+      table.increments('id').primary();
+      table.string('favorite_id');
+      table.string('favorite_name');
```

Comment:
I think we can remove "favorite" in these names since the table name will indicate that the data is related to favorites.